supplements/extra-life/infernal-machine-rebuild.xml

```diff
@@ -0,0 +1,211 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
```

Comment:
Is this a style choice or is there an actual impact? either way, Is there a way to do this easily? For some reason in VSC they all show up the same so I'm not sure how y'all are seeing this and how to easily fix it. When I hit the tab button it shows the same dots as if I just hit the space bar (only some files shows the arrows instead of dots)

Reply:
that's because your file is set to Spaces, not the Tabs.
you can change the file's indentation via this button in the lower right corner
![Tabs](https://user-images.githubusercontent.com/46818179/179303509-b9c1799f-4a07-4f6b-be0f-44f3bd29c814.png)

Reply:
> Is this a style choice or is there an actual impact?
it's mostly style choice, so all the files have the same indentation and wouldn't look weird (spaces are shorter, thus you get different length of the lines)

Reply:
here's example with `<set>` tags
![изображение](https://user-images.githubusercontent.com/46818179/179304320-7e511412-388b-40d0-87a5-d1fc06ed6520.png)